package.json

```diff
@@ -28,8 +28,9 @@
     ]
   },
   "dependencies": {
+    "@discordjs/voice": "^0.5.1",
```

Comment:
Why was this installed?

Reply:
Because something complained that it was required, though i cannot find what it was now.